rpa-builder/6.0/modules/ROOT/pages/managing-user-templates.adoc

```diff
@@ -0,0 +1,85 @@
+= Managing Templates
+
+RPA Builder enables you to save Action Steps with all settings as templates that you can reuse in the project and you can also Import and Export templates.
```

Comment:
This sentence isn't quite right because these are two separate ideas. Maybe:

RPA Builder enables you to save Action Steps and their settings as templates that you can reuse in the project. You can Import and Export these templates.